website/tutorial/basic/02-getting-started.mdx

```diff
@@ -0,0 +1,325 @@
+---
+id: 02-getting-started
+title: Getting Started
+sidebar_label: Getting Started
+---
+
+In this chapter, you will learn about the GraphQL schema:
+
+1. How it performs as an API contract between the consumer and the provider,
+2. How you can use `graphql` library as a basic GraphQL execution mechanism
+3. What is a GraphQL operation and how you can use it.
+
+### Getting Started with GraphQL
+
+To get a better understanding of how GraphQL works, you can start by [reading this tutorial about GraphQL basics](https://www.howtographql.com/basics/0-introduction/).
```

Comment:
Is this something we really think is good? we should also review that

Reply:
maybe @notrab has an alternative video for explaining GrapHQL basics? :)

Reply:
added it to https://github.com/dotansimha/graphql-yoga/issues/916